### ISSUE TITLE
docs: Cookbook Onboard (AI Assistant) integration

### ISSUE THREAD

**Issue by ClockRide** [ApeWorX/ape#2197]
### Overview/Specification

The AI assistant and co-pilot is trained on all existing Ape resources (source code, docs website, etc.) and is available as a standalone modal or as an embeddable button on any page (recommended for technical docs). It answers developer questions about using Ape Framework, acting as both an enhanced technical documentation search tool and a Solidity coding co-pilot.

Ask Cookbook AI can also access context from thousands of data sources indexed by Cookbook.dev, in addition to Ape Framework-specific data sources. This allows it to provide the best blockchain developer-focused answers available. Cookbook will assist in tuning and calibrating the AI assistant to ensure the highest quality of responses.

### Preview
https://apeworx-cookbook.vercel.app/ape/latest/userguides/quickstart.html

![image](https://github.com/user-attachments/assets/6bf57535-d3e5-4e5a-a8ad-c3fb92566903)

**Comment by antazoey** [comment]
dang!

<img width="737" alt="Screenshot 2024-08-01 at 16 36 41" src="https://github.com/user-attachments/assets/f203c7c6-f661-4a4a-978a-b835abafcb2a">

**Comment by fubuloubu** [comment]
It does seem to be a great way to audit our documentation with common questions. Is there way to set it up for us to check and see how good of a job we are doing on documentation with answering common questions? I'm not sure if re-training on even branch push is going to work, but every release does seem like something interesting

**Comment by ClockRide** [comment]
> dang!
> 
> <img alt="Screenshot 2024-08-01 at 16 36 41" width="737" src="https://private-user-images.githubusercontent.com/19540978/354389492-f203c7c6-f661-4a4a-978a-b835abafcb2a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI3MjU1ODcsIm5iZiI6MTcyMjcyNTI4NywicGF0aCI6Ii8xOTU0MDk3OC8zNTQzODk0OTItZjIwM2M3YzYtZjY2MS00YTRhLTk3OGEtYjgzNWFiYWZjYjJhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAzVDIyNDgwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzZWU5NTE2MTFhYzI2M2JiNDI5MGQwNzNhZDJiYmI5MDAxNjAwMjE3ZjhlYjQzYWNhODJhYzJhNGI3ZDc4YWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mKy9pN1cA9Gfu9sRQ-IpL6DKflENXMjSu6LhzCc3kgA">

Good catch! It seems we encountered an issue while indexing your website, so the responses weren't accurate. It should be much better now :)

**Comment by ClockRide** [comment]
> It does seem to be a great way to audit our documentation with common questions. Is there way to set it up for us to check and see how good of a job we are doing on documentation with answering common questions? I'm not sure if re-training on even branch push is going to work, but every release does seem like something interesting

We actually already have all the mentioned features in our CMS. You can review all previous conversations with users, configure data sources, and leave feedback on bot replies that it will immediately consider. We don't have re-training on every release yet, though. Currently, it auto-updates every 24 hours, and you can manually trigger an update whenever you want through the CMS we provide to all members of your team. Please contact us at hello@cookbook.dev from your corporate email or ask your colleague, Chris, to add you to the Telegram chat we have with him so we could share an invite link
